dev-haskell/pointfree/pointfree-1.1.1.9.ebuild

```
@@ -0,0 +1,33 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.0.0
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Tool for refactoring expressions into pointfree form"
+HOMEPAGE="https://hackage.haskell.org/package/pointfree"
+
+LICENSE=""	# FIXME: license unknown to cabal. Please pick it manually.
```

Hmm... I'm not sure why they didn't properly list the license in the `.cabal` file:

https://github.com/bmillwood/pointfree/blob/06843d746d50128ba1a35f04f72000cb23f940f6/pointfree.cabal#L14

It looks like their license is `MIT`, though:

https://github.com/bmillwood/pointfree/blob/master/LICENSE

```suggestion
LICENSE="MIT"
```